filters/filter_test.go

```diff
@@ -78,6 +78,18 @@ func TestFilters(t *testing.T) {
 				"foo": "omg_asdf.asdf-qwer",
 			},
 		},
+		{
+			Name: "number",
+			Labels: map[string]string{
+				"size": "100",
```

Comment:
What is the default when no unit is specified?

Reply:
bytes?

Reply:
I think it is bytes. Let me check.